linux_os/guide/services/ntp/service_timesyncd_configured/oval/shared.xml

```diff
@@ -0,0 +1,102 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("Ensure that timesyncd is enabled and configured") }}}
+    <criteria comment="Timesyncd is enabled and configured" operator="AND">
+      <criterion test_ref="{{{ rule_id }}}_test_systemd_timesyncd_service_enabled" />
```

Comment:
There is already the `service_timesyncd_enabled` rule which checks if the service is enabled. So, I would recommend to simplify this rule by removing this tests.

Reply:
reworked it as a reference to the service_enabled rule check in [0c36411](https://github.com/ComplianceAsCode/content/pull/10670/commits/0c364114f065db6ce1e2f0806241db02689b1f8c)